src/gates/parse_gates.cpp

```diff
@@ -231,13 +266,14 @@ ParseGateResult parse_gate(std::string_view str_line)
 
 void GateStreamFromFile::advance_gate()
 {
-
     ParseGateResult maybe_gate = IgnoredInstruction{};
     while(!std::holds_alternative<gates::Gate>(maybe_gate))
     {
         std::string line;
         while(!gate_file_.eof() && line.size() ==0)
         {
+            std::cout << line;
```

Comment:
I expect this to break the output when printing to stdout. I think it should be removed before merging